src/Routes.js

```diff
@@ -6,6 +6,11 @@ import Loading from './Components/Loading/Loading';
 const Register = lazy(() =>
   import(/* webpackChunkName: "Register" */ './Routes/Register/Register')
 );
+
```

Comment:
is this intended? Im not sure why we'd do this unless there was a FF

Reply:
Yeah, this feature is not complete. I'm giving it the malware treatment where I simply create a new component and we'll replace it when the feature is complete.